tests/kuttl/tests/dataplane-deploy-no-nodes-test/00-dataplane-create.yaml

```diff
@@ -12,6 +12,20 @@ metadata:
 data:
   ovsdb-config: test-ovn-config
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: neutron-ovn-metadata-agent-neutron-config
+data:
+  10-neutron-metadata.conf: dGVzdC1uZXV0cm9uLW92bi1tZXRhZGF0YS1hZ2VudC1jb25maWc=
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: nova-metadata-neutron-config
+data:
+  05-nova-metadata.conf: dGVzdC1ub3ZhLW1ldGFkYXRhLWNvbXB1dGUtY29uZmln
```

Comment:
this still point to old name i.e test-nova-metadata-compute-config instead of test-nova-metadata-neutron-config, shouldn't matter i think but just noticed.

Reply:
yeap that's what i meant, with PR update it was just missed but will not hurt as it's dummy data.